kubernetes-deployment.md

```diff
@@ -0,0 +1,60 @@
+# Kubernetes Deployment
+
+### Prerequisites
+Requires docker to be installed. We have tested our deployment against docker version ```18.09.6```
+
+### Steps
+Setup Kubernetes on your system by running the following script: 
+```
+bash k8s-deployment/dind_k8s_cluster.sh up
+```  
+1. The script downloads kubectl and sets it in ```/usr/local/bin``` PATH environment variable.  
+**NOTE:**  
+i. The script downloads kubectl version ```v1.13.0```, if you already have some other version of kubectl downloaded keep a backup of it.  
+ii. Ensure ```/usr/local/bin``` is set in your PATH environment variable as this is where the script places your kubectl.  
+2. The script sets up Docker in Docker(dind) version ```v1.13``` on your system. If you want to be running dind using some alternate version, follow steps given in this [link](https://github.com/kubernetes-sigs/kubeadm-dind-cluster).   
```

Comment:
IMO, It should look like :
"The script sets up Kubernetes cluster (Version v1.13) based on DIND(Docker in Docker). An Alternate Kubernetes version can be sets up by following steps in [link](https://github.com/kubernetes-sigs/kubeadm-dind-cluster)"

Reply:
Modified as per suggestion.